build/cli.ts

```diff
@@ -241,6 +241,7 @@ async function buildDocuments(
     }
 
     const hash = crypto.createHash("sha256").update(docString).digest("hex");
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
Since these both are related to ignored parameters, it's also possilbe one of the other settings might be used for the global rule https://eslint.org/docs/latest/rules/no-unused-vars#argsignorepattern but since it's pretty uncommon, disables might be better